tsdb/engine/wal/wal.go

```diff
@@ -74,6 +74,9 @@ const (
 	// MetaFileExtension is the file extension for the log files of new fields and measurements that get created
 	MetaFileExtension = "meta"
 
+	// CompactionExtension is the file extension we expect for compaction files
+	CompactionExtension = "CPT"
```

Comment:
I don't really care. It's a temporary file anyway so it only exists momentarily. I say just leave it.